libs/wire-api/src/Wire/API/Error/Galley.hs

```diff
@@ -225,3 +264,21 @@ instance Member (Error DynError) r => ServerEffect (Error TeamFeatureError) r wh
     LegalHoldWhitelistedOnly -> dynError @(MapError 'LegalHoldWhitelistedOnly)
     DisableSsoNotImplemented -> dynError @(MapError 'DisableSsoNotImplemented)
     FeatureLocked -> dynError @(MapError 'FeatureLocked)
+
+--------------------------------------------------------------------------------
+-- Proposal failure
+
+data ProposalFailure = ProposalFailure
+  { pfInner :: Wai.Error
+  }
+
+type instance ErrorEffect ProposalFailure = Error ProposalFailure
+
+-- Proposal failures are only reported generically in Swagger
+instance IsSwaggerError ProposalFailure where
+  addToSwagger = S.allOperations . S.description %~ Just . (<> desc) . fold
+    where
+      desc = "\n\n!!PROPOSAL FAILURE!!" -- TODO
```

Comment:
Have you addressed the TODO in a later commit?

Reply:
Yes.